pkgs/tools/filesystems/fuse-archive/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fuse
+, libarchive
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fuse-archive";
+  version = "0.1.14";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "fuse-archive";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-l4tIK157Qo4m611etwMSk564+eC28x4RbmjX3J57/7Q=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    fuse
+    libarchive
+  ];
+
+  env.NIX_CFLAGS_COMPILE = "-D_FILE_OFFSET_BITS=64";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp out/fuse-archive $out/bin/fuse-archive
+
+    runHook postInstall
+  '';
```

I was reading the [Makefile](https://github.com/google/fuse-archive/blob/v0.1.14/Makefile) and found a `prefix` variable. Can you try this?

```suggestion
  makeFlags = [
    "prefix=${placeholder "out"}" 
  ];
```

---

That worked as well, updated.